superset/assets/src/welcome/App.jsx

```diff
@@ -16,42 +16,47 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React from 'react';
-import { hot } from 'react-hot-loader';
-import thunk from 'redux-thunk';
-import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
-import { Provider } from 'react-redux';
+import React from "react";
+import { hot } from "react-hot-loader";
+import thunk from "redux-thunk";
+import { createStore, applyMiddleware, compose, combineReducers } from "redux";
+import { Provider } from "react-redux";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
-import messageToastReducer from '../messageToasts/reducers';
-import { initEnhancer } from '../reduxUtils';
-import setupApp from '../setup/setupApp';
-import Welcome from './Welcome';
-import Menu from '../components/Menu/Menu';
+import messageToastReducer from "../messageToasts/reducers";
+import { initEnhancer } from "../reduxUtils";
+import setupApp from "../setup/setupApp";
+import Welcome from "./Welcome";
+import Menu from "../components/Menu/Menu";
```

Comment:
Looks like prettier auto formatted this for me. I'll rerun once https://github.com/apache/incubator-superset/pull/8648 lands and the desired config is available.